sawmill-core/src/main/java/io/logz/sawmill/processors/DateProcessor.java

```diff
@@ -202,12 +208,7 @@ private ZonedDateTime getZonedDateTime(String value, DateTimeFormatter formatter
     private ZonedDateTime getUnixDateTime(Long value) {
         long unixTimestamp = value;
         Instant instant;
```

Comment:
Now redundant

Reply:
this instant is necessery to created ZonedDatetime....